### PR TITLE
Fix #408, Remove device driver references in code

### LINKED ISSUE #408: Remove device driver references from code
**Is your feature request related to a problem? Please describe.**
#377 is to remove device driver requirements, there are also references in the software.

**Describe the solution you'd like**
Remove references to device drivers, "CFE_DRV", etc from software, configuration files, documentation, etc.

For example:
https://github.com/nasa/cFE/blob/5d7d766941369b6e92a0328217e43d5fd02a8cf1/fsw/cfe-core/src/es/cfe_es_apps.c#L338-L342

**Describe alternatives you've considered**
None

**Additional context**
None

**Requester Info**
Jacob Hageman - NASA/GSFC

### PR DESCRIPTION
**Pull request by avan989**
**Describe the contribution**
reference pull request https://github.com/nasa/cFE/pull/475

update remove reference to driver

**Contributor Info - All information REQUIRED for consideration of pull request**
Anh Van, NASA Goddard

### CODE REVIEW

**Comment by avan989** [comment]
accidentally closed

**Comment by avan989** [comment]
updated.

File: docs/src/cfe_es.dox [hunk 1]
```diff
@@ -975,4 +974,3 @@
   The following are configuration parameters used to configure the cFE Executive Services
   either for each platform or for a mission as a whole.
 **/
-
```
Comment (skliper) [inline docs/src/cfe_es.dox]: Whitespace wierdness?

File: fsw/cfe-core/unit-test/es_UT.c [hunk 2]
```diff
@@ -7091,5 +7062,4 @@ void TestStaticApp(void)
               "CFE_ES_StartStaticApplications",
               "*Not implemented* Unexpected entry type");
 }
-#endif
-
+#endif
```
Comment (skliper) [inline fsw/cfe-core/unit-test/es_UT.c]: Endline weirdness?

File: fsw/cfe-core/unit-test/es_UT.h [hunk 3]
```diff
@@ -364,4 +363,4 @@ void TestStaticApp(void);
 #endif
 
 
-#endif /* _es_ut_h_ */
+#endif /* _es_ut_h_ */
```
Comment (skliper) [inline fsw/cfe-core/unit-test/es_UT.h]: Endline weirdness?

**Comment by avan989** [comment]
updated.

**Comment by astrogeco** [comment]
CCB 20200212 - Approved

**Pull request merged**